file_encrypt.routing.yml

```diff
@@ -0,0 +1,7 @@
+file_encrypt.file_download:
+  path: '/encrypt/files/{scheme}'
+  defaults:
+    _controller: \Drupal\system\FileDownloadController::download
```

Comment:
This controller is driven by the "scheme" below, which uses the encryption stream?